CONTRIBUTORS.md

```diff
@@ -0,0 +1,3 @@
+# Contributors
+
+- [Felix Junghans](https://github.com/felixjunghans) - Created an issue in FlutterPlaza/no_screenshot that received 4 comments
```

Comment:
Does the fact that his issue received four comments add value to his contribution?
If not I believe that part should be left out.